curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iv---put-method.english.md

```diff
@@ -13,11 +13,8 @@ This exercise is similar to the preceding. Look at it for the details.
 
 ## Instructions
 <section id='instructions'>
-Send <br>
-<blockquote>
-{<br>
-  &nbsp;surname: 'da Verrazzano'<br>
-}</blockquote>. Replace <code>assert.fail()</code> and make the test pass.
+Send <code>{ surname: 'da Verrazzano' }.
```

Comment:
```suggestion
Send 

```json
{ 
  "surname": "da Verrazzano"
}
```
.
```

Reply:
Resolved in new commit.